pkg/engine/armvariables.go

```diff
@@ -60,15 +60,15 @@ func getK8sMasterVars(cs *api.ContainerService) (map[string]interface{}, error)
 	masterProfile := cs.Properties.MasterProfile
 	profiles := cs.Properties.AgentPoolProfiles
 
-	var useManagedIdentity, userAssignedID, userAssignedClientID, enableEncryptionWithExternalKms bool
+	var useManagedIdentity, userAssignedID, userAssignedClientID, createKeyVault bool
```

Comment:
@ritazh to sanity-check this re-use of the existing `clusterKeyVaultName` variable originally created for external KMS feature (lgtm)

Reply:
@ritazh can you please review?